### PR TITLE
Remove team assignments from task

### LINKED ISSUE #1279: Remove team assignment from tasks
Since tasks are required to belong to a project and a project itself is assigned to a team, it is unecessary and possibly confusing to have a team assignment on the task level as well.

<a href="https://timer.scm.io/repos/537237ce1a00001a003c4a14/issues/1279/create?referer=github" target="_blank">Log Time</a>

### PR DESCRIPTION
**Pull request by rschwanhold**
### Mailable description of changes:
 - Tasks no longer have the field 'team'. 
 - The linkage between Task and Team is implemented through Project. (The Task knows the Project and the Projects knows the Team)
 - This leads to the following changes:
   The creation on Tasks does not require the name of the Team anymore.
   An example for Bulk-Creation now looks like this:
   `ROI2017_wkw, 5afeb79073000071006904a5, sampleExp, 1, 512,512,512, 0, 0, 0, 1, 500, 500, 500, 100, 100, 100, sampleProject`
    instead of 
`ROI2017_wkw, 5afeb79073000071006904a5, sampleExp, 1, 512,512,512, 0, 0, 0, 1, Connectomics department, 500, 500, 500, 100, 100, 100, sampleProject
`

### Steps to test:
test if the basic functions for 'task' are still working
- create a new task
- press the 'Get a new Task'-Button

test the project_list_view:
- go to 'Administration' -> 'Projects' and verify that the names of the projects are shown

### Issues:
- fixes #1279 

------
- [x] Ready for review

### CODE REVIEW

File: app/controllers/TaskController.scala [hunk 1]
```diff
@@ -91,8 +89,7 @@ class TaskController @Inject() (val messagesApi: MessagesApi)
       params <- JsonHelper.parseJsonToFox[NmlTaskParameters](jsonString) ?~> Messages("task.create.failed")
       taskType <- TaskTypeDAO.findOneById(params.taskTypeId) ?~> Messages("taskType.notFound")
       project <- ProjectDAO.findOneByName(params.projectName) ?~> Messages("project.notFound", params.projectName)
-      team <- TeamDAO.findOneByName(params.teamName) ?~> Messages("team.notFound")
-      _ <- ensureTeamAdministration(request.identity, team._id)
+      _ <- ensureTeamAdministration(request.identity, project._team) //ensureTeamAdministration(request.identity, teamBSON)
```
Comment (fm3) [inline app/controllers/TaskController.scala]: is this a resolved todo?

File: conf/evolutions_postgres/006-remove-team-assignment-from-task.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+CREATE VIEW webknossos.tasks_ AS SELECT * FROM webknossos.tasks WHERE NOT isDeleted;
+COMMIT TRANSACTION;
+
+
+-- DOWN:
+
+
+START TRANSACTION;
+DROP VIEW webknossos.tasks_;
+ALTER TABLE webknossos.tasks ADD COLUMN _team CHAR(255); -- which data type is 'team'?
```
Comment (fm3) [inline conf/evolutions_postgres/006-remove-team-assignment-from-task.sql]: should be CHAR(24) (that’s our id type, fits a BSONObjectId)

**Comment by rschwanhold** [comment]
When merging this PR, we also need to update the example of Bulk-Creation in the wiki of webknossos-wiki to e.g. `ROI2017_wkw, 5afeb79073000071006904a5, sampleExp, 1, 512,512,512, 0, 0, 0, 1, 500, 500, 500, 100, 100, 100, sampleProject`

**Comment by rschwanhold** [comment]
@fm3 can you test this again?

**Pull request merged**